doc/radosgw/config-ref.rst

```diff
@@ -562,6 +582,35 @@ Swift Settings
 :Default: ``auth``
 
 
+``rgw swift account in url``
+
+:Description: Whether or not the Swift account name should be included
+              in the Swift API URL.
+
+              If set to ``false`` (the default), then the Swift API
+              will listen on a URL formed like
+              ``http://host:port/<rgw_swift_url_prefix/v1``, and the
```

Comment:
@fghaas note that the `rgw_swift_url_prefix` here and on other places is missing `>`

Reply:
@yehudasa Thanks! Well spotted. Fixed, amended, and force-pushed. Since this has been in the making for a while now, should I be rebasing on current master?